microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1340,7 +1356,7 @@ See also:
 * `CoreWebView2` Class:
    * [CoreWebView2.ContentLoading Event](/dotnet/api/microsoft.web.webview2.core.corewebview2.contentloading)
    * [CoreWebView2.DOMContentLoaded Event](/dotnet/api/microsoft.web.webview2.core.corewebview2.domcontentloaded)
-   * [CoreWebView2.FrameNavigationCompleted Event](/dotnet/api/microsoft.web.webview2.core.corewebview2.framenavigationcompleted) - superseded; use `CoreWebView2Frame.NavigationCompleted` instead<!--todo: tech review this added note; confirm this API is superseded-->
+   * [CoreWebView2.FrameNavigationCompleted Event](/dotnet/api/microsoft.web.webview2.core.corewebview2.framenavigationcompleted) - superseded; use `CoreWebView2Frame.NavigationCompleted` instead<!-- todo: tech review this added note; confirm this API is superseded-->
```

Comment:
@peiche-jessica @novac42 - is it true that this API is superseded?

goal: remove/resolve old todo's

Reply:
I believe so since the CoreWebView2Frame class supports more scenarios to allow interactions with frames. This one can be resolved unless Jessica says otherwise.

Reply:
removed the todo comment x3